eng/targets/Npm.Common.targets

```diff
@@ -11,6 +11,7 @@
     <IntermediateOutputPath>$([MSBuild]::NormalizeDirectory('$(BaseIntermediateOutputPath)'))$(Configuration)\</IntermediateOutputPath>
     <InstallArgs Condition="'$(RestoreLockedMode)' == 'true'">$(InstallArgs) --frozen-lockfile</InstallArgs>
     <InstallArgs>$(InstallArgs) --ignore-engines</InstallArgs>
+    <InstallArgs Condition=" '$(OS)' == 'Windows_NT' ">$(InstallArgs) --python=C:\python3.7.0\python.exe</InstallArgs>
```

Comment:
Will want to add an exists check on this to ensure things continue to run locally even if this doesn't exist.

Reply:
good call and wouldn't merge without it;   I'm still at the "get something to work" stage though.